src/frontend/src/lib/services/address.services.ts

```diff
@@ -69,22 +72,43 @@ const loadTokenAddress = async <T extends Address>({
 	return { success: true };
 };
 
+// We use the Testnet address for Regtest.
+type BtcTokenIdsWithAddresses = typeof BTC_MAINNET_TOKEN_ID | typeof BTC_TESTNET_TOKEN_ID;
+type BtcNetworkWithAddresses = Exclude<BitcoinNetwork, 'regtest'>;
```

Comment:
For the naming, I would ask suggestions to @peterpeterparker , usually he gets them

Reply:
Done!